test/scenarios/bookshop/orderBy.test.js

```diff
@@ -0,0 +1,46 @@
+const cds = require('../../cds.js')
+const bookshop = require('path').resolve(__dirname, '../../bookshop')
+
+describe('Bookshop - Order By', () => {
+  const { expect } = cds.test(bookshop)
+
+  test('collations', async () => {
+    // the original query is sorted by the **column** "createdBy"
+    // the resulting query has two query sources in the end --> Authors and Books
+    // even though both Tables have the element "createdBy", the DB should be able to resolve the
+    // order by reference to the column unambiguously
+    const query = CQL(`SELECT from sap.capire.bookshop.Books {
+      createdBy,
+      author.name as author
+    } order by createdBy`)
+    const res = await cds.run(query)
+    expect(res.length).to.be.eq(5)
```

Comment:
I would recommend putting a limit on the query and expecting `1`. As it is order relevant it might make sense to check for the returned value. As currently it just checks that it does not throw.